src/constants/rdf_predicates.js

```diff
@@ -2,7 +2,11 @@ import { SCHEMA_INRUPT } from '@inrupt/vocab-common-rdf';
 
 const RDF_PREDICATES = {
   ...SCHEMA_INRUPT,
-  uploadDate: 'https://schema.org/uploadDate'
+  uploadDate: 'https://schema.org/uploadDate',
+  title: 'https://schema.org/title',
+  message: 'https://schema.org/Message',
+  recipient: 'https://schema.org/recipient',
+  sender: 'https://schema.org/sender'
```

Comment:
Can possibly be refactored in a future PR. But for now, it'll do.

Reply:
yeah destructure in future?

Reply:
Yeah probably.